cowprotocol/cowswap
# feat(wallet-connect-2): lazy loading for wc1 and wc2

# Summary

The key difference here is that `@walletconnect` bundle loads only on demand, before this fix it was loaded always.
The result: -238kb from page loading.

Before:
<img width="906" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/169d47e9-a2cf-4c2a-864b-676e16923903">


After:
<img width="977" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/2df88b77-42bd-42b5-a740-c944c98f37bc">


  # To Test

1. Wallet connect v1
 - connect wallet (all networks)
 - place order
 - reload page with connected wallet (should restore connection)
2. Wallet connect v2
 - connect wallet (all networks)
 - place order
  - reload page with connected wallet (should restore connection)
